dev/devfile.yaml

```diff
@@ -17,19 +17,28 @@ components:
     memoryLimit: 1512Mi
     type: dockerimage
     alias: appsodyrun
-    image: 'ajymau/java-openliberty-dev:latest'
+    image: 'kabanero/ubi8-maven:0.3.1'
     env:
       - value: TEST2
         name: MODE2
       - value: myval2
         name: myprop2
+    volumes:
+      - name: m2
+        containerPath: /root/.m2/repository
 apiVersion: 1.0.0
 commands:
+  - name: devInit
+    actions:
+      - workdir: /projects/user-app
+        type: exec
+        command: mvn -B -DskipTests=true liberty:install-server liberty:create liberty:install-feature install dependency:go-offline 
```

Comment:
Hmm...not sure what this command should be.   Running "install" is probably too much.. it will run ITs..what exactly should and shouldn't we run?  

Why don't we stick to pure caching + Liberty install (as a cache)..so maybe more like:

  `mvn -B liberty:install-server liberty:create liberty:install-feature dependency:go-offline`